src/Renderer/Shader/PointsFS.glsl

```diff
@@ -18,9 +18,12 @@ void main() {
 
 // Early discard (clipping planes and shape)
 #include <clipping_planes_pars_fragment>
+    if (vColor.a == 0.0) {
+        discard;
```

Comment:
Should we keep the discard when ```opacity === 0``` or ```visible === false```.
This i can't tell, i don't have enought knowledge about shader. WDYT @Desplandis @jailln

Reply:
Yeah, we should drop this condition and respect material semantics which specify that a material with `transparent = false` shall be opaque.

Reply:
Removed